### DISCUSSION TITLE
got gets stuck when performing a HEAD request on URL that returns malformed response

### DISCUSSION THREAD

**Discussion by JahzVH** [sindresorhus/got#1728] [Q&A] [answered]
Hi, 
I think I encountered the same problem as described on issue #1527, I send a HEAD request and I never receive any response or error, the process just hangs there.

I see a fix has been done(77df9c33db5ba3126f54317171e1cfcfceefc3d5), but it has not been released yet right?
Is there any plan to release this fix?

Thank you!

Node v12
got v11.8.2
URL: https://mp.weixin.qq.com/s/L1jFI72W0RHzSaf08qKzLg

**Comment by sindresorhus** [comment, marked as answer]
> Is there any plan to release this fix?

Yes, when it's done. There are still some things that needs to be done.

---

For now, you can install the latest commit: `npm install "sindresorhus/got#62305d77d3428b5c714d21b4bbee68cc75b5f787"`

**Comment by JahzVH** [comment]
Thank you @sindresorhus.
Will use this while waiting for the next release.
